Makefile

```diff
@@ -35,7 +35,13 @@ install-addlicense:
 install-lint:
 	(which $(GOPATH)/bin/golangci-lint || go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.40.1)
 
-generate: install-stringer
+install-deepcopy-gen:
```

Comment:
I suggest we move `.PHONY` to each target individually otherwise it's easy to forget about it when creating targets. Doesn't have to be in this PR.

Reply:
looks like we're missing quite a few already. So I'll make a separate PR for that.